fdbserver/include/fdbserver/DataDistribution.actor.h

```diff
@@ -583,8 +583,18 @@ class BulkLoadTaskCollection : public ReferenceCounted<BulkLoadTaskCollection> {
 public:
 	BulkLoadTaskCollection(UID ddId) : ddId(ddId) { bulkLoadTaskMap.insert(allKeys, Optional<DDBulkLoadEngineTask>()); }
 
-	// Return true if there exists a bulk load task
-	bool overlappingTask(KeyRange range) {
+	// Return true if there exists a bulk load job/task or the collection has not been initialized.
+	// This takes effect only when DDBulkLoad Mode is enabled.
+	bool onBulkLoading(KeyRange range) {
```

Comment:
s/onBulkLoading/bulkLoading/ as in, we are actively running a bulk load (If that is what you intend)